buildSrc/src/main/java/com/yelp/gradle/bento/GlobalDependencies.kt

```diff
@@ -9,33 +9,37 @@ object Publishing {
 }
 
 object Versions {
-    const val COMPILE_SDK = 33
+    const val COMPILE_SDK = 34
     const val TARGET_SDK = 33
     const val MIN_SDK = 21
 
+    // Java
+    const val SOURCE_COMPATIBILITY = 11
+    const val TARGET_COMPATIBILITY = 11
```

Comment:
Can you make sure to declare these updates in the change log: https://github.com/Yelp/bento/blob/master/CHANGELOG.md#-breaking-changes

Reply:
i updated the commit message to include `!` in order to be marked as a breaking change in the change log 
https://www.conventionalcommits.org/en/v1.0.0/